var/spack/repos/builtin/packages/hdf5-vol-log/package.py

```diff
@@ -18,12 +18,10 @@ class Hdf5VolLog(AutotoolsPackage):
     version("master-1.1", branch="master")
 
     version("1.4.0", tag="logvol.1.4.0")
-    version("1.3.0", tag="logvol.1.3.0")
```

Comment:
Like the ones above, I think all of the older versions may need to be dropped.

Reply:
@lrknox

Reply:
Dropped versions 1.2.0 and 1.1.0.